web/src/js/__tests__/urlStateSpec.js

```diff
@@ -0,0 +1,79 @@
+import initialize from '../urlState'
+
+import reduceFlows from '../ducks/flows'
+import reduceUI from '../ducks/ui/index'
+import reduceEventLog from '../ducks/eventLog'
+import * as flowsActions from '../ducks/flows'
+
+import configureStore from 'redux-mock-store'
+
+const mockStore = configureStore()
+
+describe('updateStoreFromUrl', () => {
+    history.replaceState = jest.fn()
```

Comment:
Ah, okay. Missed that part. :-)
How about we just export `updateStoreFromUrl` and `updateUrlFromStore`? 😁